src/getOptions.js

```diff
@@ -49,6 +49,9 @@ export default loaderContext => {
     const options = getOptions(loaderContext) || {};
 
     options.localeInterpolate = getLocaleInterpolate(options);
+    if (options.requireAngular === true || options.requireAngular === false) {
```

Comment:
@Ardeshir81 You don't need it, because you can override default options:

```js
module.exports = {
    module: {
        rules: [{
            test: /\.json$/,
            loader: 'angular-translate-loader',
            options: {
                requireAngular: false
            }
        }]
    }
};
```